homeassistant/components/velux/config_flow.py

```diff
@@ -0,0 +1,108 @@
+"""Config flow for Velux integration."""
+from typing import Any
+
+from pyvlx import PyVLX, PyVLXException
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PASSWORD
+from homeassistant.core import DOMAIN as HOMEASSISTANT_DOMAIN
+from homeassistant.data_entry_flow import FlowResult
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.issue_registry import IssueSeverity, async_create_issue
+
+from .const import DOMAIN, LOGGER
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+    }
+)
+
+
+class VeluxConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for velux."""
+
+    async def async_step_import(self, config: dict[str, Any]) -> FlowResult:
+        """Import a config entry."""
+        self._async_abort_entries_match({CONF_HOST: config[CONF_HOST]})
+
+        def create_repair(error: str | None = None) -> None:
+            if error:
+                async_create_issue(
+                    self.hass,
+                    DOMAIN,
+                    f"deprecated_yaml_import_issue_${error}",
+                    breaks_in_ha_version="2024.9.0",
+                    is_fixable=False,
+                    issue_domain=DOMAIN,
+                    severity=IssueSeverity.WARNING,
+                    translation_key=f"deprecated_yaml_import_issue_${error}",
```

Comment:
```suggestion
                    f"deprecated_yaml_import_issue_{error}",
                    breaks_in_ha_version="2024.9.0",
                    is_fixable=False,
                    issue_domain=DOMAIN,
                    severity=IssueSeverity.WARNING,
                    translation_key=f"deprecated_yaml_import_issue_{error}",
```

That's javascript syntax :)